config/bundles.php

```diff
@@ -15,4 +15,5 @@
     Symfony\Bundle\WebServerBundle\WebServerBundle::class => ['dev' => true],
     Symfony\Bundle\WebProfilerBundle\WebProfilerBundle::class => ['dev' => true, 'test' => true],
     DAMA\DoctrineTestBundle\DAMADoctrineTestBundle::class => ['test' => true],
+    Doctrine\Bundle\MigrationsBundle\DoctrineMigrationsBundle::class => ['all' => true],
```

Comment:
will it be worth add a bundle that we don't use? is there a way to avoid/skip the installation of a transitive package through composer?

Reply:
We don't use it (yet) but I think it's better to be 100% in sync with Symfony recipes.